docs/api/extending_the_rest_api.md

```diff
@@ -223,20 +251,45 @@ class Greetings extends BaseParser
     }
 }
 ```
+You should then add a new method to the previous `DefaultController` to handle the new post request:
+```
+use eZ\Publish\Core\REST\Common\Message;
+//...
+    public function sayHelloUsingPost()
+    {
 
-**My/Bundle/RestBundle/Resources/config/services.yml**
+      $createStruct = $this->inputDispatcher->parse(
+          new Message(
+              array('Content-Type' => $request->headers->get('Content-Type')),
+              $request->getContent()
+          )
+      );
+
+      $name =  $createStruct->name ;
+
+      //...
+
+    }
 
+```
+The `inputDispatcher` is responsible to match the `Content-Type` sent in the header with the Greetings InputParser class.
+
+Finally, a new Route should be added to routing_rest.yml
 ``` yaml
-services:
-    myRestBundle.controller.default:
-        class: My\Bundle\RestBundle\Rest\Controller\Default
-        parent: ezpublish_rest.controller.base
+myRestBundle_hello_world_using_post:
+    path: /my_rest_bundle/hello/
+    defaults:
+        _controller: myRestBundle.controller.default:sayHelloUsingPost
+    methods: [POST]
 ```
+<div class="admonition note">
```

Comment:
We have an easier to read way of placing note boxes directly in markdown. In this case it would be:

```
!!! note

    Post Requests are not able to access the repository without performing a user authentication. More information [REST API Authentication ](https://github.com/ezsystems/ezpublish-kernel/blob/master/doc/specifications/rest/REST-API-V2.rst#authentication).
```

(note the indent in the first line of the actual text)